src/main/java/thrift/logic/parser/ParserUtil.java

```diff
@@ -28,6 +30,7 @@
 
     public static final String MESSAGE_INVALID_INDEX = "Index is not a non-zero unsigned integer.";
     public static final String MESSAGE_INVALID_CURRENCY = "Currency entered is not valid. (Must exist)";
+    public static final String MESSAGE_INVALID_MONTH_FORMAT = "Invalid month format!";
```

Comment:
No need to re-declare it if it exists as a static variable `ListCommandParser` (see first comment above).

Reply:
declared in LCP and removed from parserutil.